plugins/google_analytics/girder_google_analytics/web_client/main.js

```diff
@@ -5,21 +5,24 @@ import './lib/backbone.analytics';
 import './routes';
 
 events.on('g:appload.after', function () {
-    /* eslint-disable */
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-    /* eslint-enable */
+    var dnt = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
```

Comment:
`navigator.msDoNotTrack` [is only used in IE < 11](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack), which Girder doesn't support (and will not even load Girder) anyway. It would be safe to remove that vendor-specific case.

Reply:
Ok, removed.